src/vip-framework/index.ts

```diff
@@ -76,6 +76,11 @@ const executeCommand = async (timelock: SignerWithAddress, proposal: Proposal, c
   if (network.zksync && feeData.maxFeePerGas) {
     // Sometimes the gas estimation is wrong with zksync
     txnParams.maxFeePerGas = feeData.maxFeePerGas.mul(15).div(10);
+    // Increase gas limit for pool registry commands in simulations. Sometimes it is estimated too low in complex transactions.
```

Comment:
Cannot we define this out of the framework? It seems too specific to be included to something that should be as much generic as possible